src/tests/mem/Mybuild

```diff
@@ -47,3 +55,6 @@ module mmap {
 
 	depends embox.mem.vmem
 }
+
```

Comment:
Extra newlines, remove, please.

Reply:
I would recommend looking in "uncrustify" job output 

https://travis-ci.org/embox/embox/jobs/403160384

(can be reached from PR travis checks https://travis-ci.org/embox/embox/builds/403160369)

It suggests diff that should fix codestyle (as it understand it), so It's good for quick look for possible issues. But it should be taken with grain of scepsis, sometime it recommend strangely bad changes :)